Libs/DICOM/Core/Resources/dicom-schema.sql

```diff
@@ -84,7 +85,7 @@ CREATE TABLE 'Series' (
   'DisplayedFieldsUpdatedTimestamp' DATETIME NULL ,
   PRIMARY KEY ('SeriesInstanceUID') );
 
-CREATE UNIQUE INDEX IF NOT EXISTS 'ImagesFilenameIndex' ON 'Images' ('Filename');
```

Comment:
Should the URL be indexed as well ?

Reply:
Hmmm, maybe, since that is used in the `fileValue` method to find the `instanceUID` so it can be used to look up values in the tag cache. Bypassing the tag cache is still probably better, but an index on this might have addressed some of the performance issues I was seeing.